src/cftlib/resources/docker/cft-lib-doc-stack.yml

```diff
@@ -22,7 +22,7 @@ services:
       LOGGING_LEVEL_UK_GOV_HMCTS_IDAM: WARN
       IDAM_USER_BASE_URI: "${IDAM_STUB_SERVICE_NAME:-http://rse-idam-simulator:5062}"
       IDAM_S2S_BASE_URI: http://host.docker.internal:4502
-      STORAGEACCOUNT_PRIMARY_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azure-storage-emulator-azurite:10000/devstoreaccount1
+      STORAGEACCOUNT_PRIMARY_CONNECTION_STRING: "${STORAGEACCOUNT_PRIMARY_CONNECTION_STRING:DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=${Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==};BlobEndpoint=http://azure-storage-emulator-azurite:10000/devstoreaccount1}"
```

Comment:
looks as if there some extra characters that got added there to the connection string

Reply:
but that's gonna mess up the connection string, I think